src/SkyHope.LibraryManager.WebApi/SkyHope.LibraryManager.WebApi.http

```diff
@@ -0,0 +1,6 @@
+@SkyHope.LibraryManager.WebApi_HostAddress = http://localhost:5238
```

Comment:
You don't really need this file. It looks out of date anyway.
You can read more about what it is here:
https://www.rahulpnath.com/blog/http-files-asp-net-core-dotnet/